client/src/modules/list/components/Items/ListItem/ListItem.jsx

```diff
@@ -30,6 +30,8 @@ import ListItemDescription from '../ListItemDescription';
 import { ItemStatusType } from '../model/actionTypes';
 
 class ListItem extends PureComponent {
+  isMounted = false;
```

Comment:
that is huge antipattern and we should avoid it: https://reactjs.org/blog/2015/12/16/ismounted-antipattern.html

Try to move the sockets up - to the list itself, not to every item and create bool property for list item like `blocked` and set it from list.

This should be simple component and now it is infected with sockets ;)